src/components/list/list.scss

```diff
@@ -6,6 +6,8 @@
 
```

Comment:
Comment on the commit messages:
- `feat: add mixing for flat clickable items`
  This one is actually a `chore` imo, because the mixin isn't exported, so this commit alone has no effect for the consumer of lime-elements. (nitpick: typo "mixing -> mixin")
- `chore: import mixins file in lists scss`
  This one I think should be combined with the next one.
- feat(list): make list items more distinct & interactive
  Perfect!